### PR TITLE
Ft buses

### PR DESCRIPTION
**Pull request by Yassin-hagenimana**

### CODE REVIEW

File: src/modules/buses/controller/busesController.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        status: "fail",
+        message: "Bus with that ID not found",
+      });
+    }
+
+    res.status(200).json({
+      status: "success",
+      data: {
+        bus,
+      },
```
Comment (yvettegahamanyi) [inline src/modules/buses/controller/busesController.ts]: I think here you were supposed to use the general response format that we are all using 
for example for the first response
` responseUtil.handleError(BAD_REQUEST, "Bus with that ID not found");
            return responseUtil.response(res);`

File: src/modules/buses/controller/busesController.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    res.status(200).json({
+      status: "success",
+      results: buses.length,
+      buses,
+    });
+  } catch (error: any) {
+    res.status(500).json({
+      status: "error",
+      message: error.message,
+    });
```
Comment (yvettegahamanyi) [inline src/modules/buses/controller/busesController.ts]: same here(response format)

**Comment by Yassin-hagenimana** [comment]
Changed to ft-deploy

**Review by yvettegahamanyi — COMMENTED** [review]
I think you are missing some changes from develop

**Pull request closed without merging**